assets/src/edit-story/components/tabview/index.js

```diff
@@ -233,7 +236,13 @@ function TabView({
       {tabs.map(({ id, title, icon: Icon, ...tabRest }) => (
         <Tab
           key={id}
-          ref={(tabRef) => (tabRefs.current[id] = tabRef)}
+          ref={(tabRef) => {
+            if (tabRefs) {
+              return (tabRefs[id].current = tabRef);
+            }
+
+            return (internalTabRefs.current[id] = tabRef);
+          }}
```

Comment:
nit: now that this is using brackets I prefer an assignment, then a return instead of the `return (assignment)` syntax

Reply:
oh yeah, good shout. I don't think we even need to return anything. I'll fix this